firstvoices/backend/tests/test_apis/base_api_test.py

```diff
@@ -459,11 +459,7 @@ class DestroyApiTestMixin:
 
     @pytest.mark.django_db
```

Comment:
I changed the base delete tests to use a superadmin, since not all models are editable by a language admin